dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/CRUD/CollectionConformanceTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using VectorDataSpecificationTests.Models;
+using VectorDataSpecificationTests.Support;
+using VectorDataSpecificationTests.Xunit;
+using Xunit;
+
+namespace VectorDataSpecificationTests.CRUD;
+
+public class CollectionConformanceTests<TKey>(VectorStoreFixture fixture)
```

Comment:
nit: collection management generally isn't considered CRUD (the latter usually refers to operations on the rows/records inside).